mysql/check.py

```diff
@@ -322,6 +322,8 @@ def _get_config(self, instance):
         password = str(instance.get('pass', ''))
         tags = instance.get('tags', [])
         options = instance.get('options', {})
+        if options is None:
```

Comment:
`options = instance.get('options', {}) or {}` would be more pythonic. Also a comment to explain why options might be `None` would be nice I think.